src/npe2/cli.py

```diff
@@ -212,9 +215,21 @@ def _make_rows(pm_dict: dict, normed_fields: Sequence[str]) -> Iterator[List]:
             # extact nested fields
             if not val and "." in field:
                 parts = field.split(".")
-                val = info
-                while parts and hasattr(val, "__getitem__"):
-                    val = val[parts.pop(0)]
+                val = [info]
+                while parts:
+                    part = parts.pop(0)
+                    new_val = []
+                    for cur_val in val:
+                        if hasattr(cur_val, "__getitem__"):
+                            # list if overridden by function name
+                            if isinstance(cur_val[part], type([])):
+                                new_val.extend(cur_val[part])
+                            else:
+                                new_val.append(cur_val[part])
+                    val = new_val
```

Comment:
This does not look like reproduce and extend of the previous logic. It may break tests.